### ISSUE TITLE
Remove "ecmaVersion": 8 from .eslintrc.json?

### ISSUE THREAD

**Issue by NilsEnevoldsen** [NilsEnevoldsen/ScryfallLinks#50]
I think this extension isn't supposed to use features beyond ES3 in order to be compatible with ResourceLoader. https://phabricator.wikimedia.org/T75714

https://github.com/NilsEnevoldsen/ScryfallLinks/blob/92d72d279a0701206a80011c672d7d5c3e4d411c/.eslintrc.json#L11-L13

I used a higher version because I didn't know better and it appears to work. I'm going to wontfix this issue, but keep it open for informational purposes, or in case some contributor is ambitious.

**Comment by Aspallar** [comment]
I would agree with the wontfix - the ES3 limitation appears to only apply to js in a wiki's Common.js and in users personal js pages. So the only reason to re-code to ES3 would be to support Internet Explorer 11, a dying browser.
